spec/expeditor/current_thread_function_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'spec_helper'
+
+describe Expeditor::Command do
+  let(:error_in_command) { Class.new(StandardError) }
+
+  describe '#start' do
+    context 'with normal' do
+      it 'should execute on current thread' do
```

Comment:
**should** is unnecessary description. `it 'executes on current thread'` is enough.